production/loki-mixin/dashboards/loki-reads.libsonnet

```diff
@@ -20,10 +21,10 @@ local utils = import 'mixin-utils/utils.libsonnet';
 
                          matchers:: {
                            cortexgateway: [utils.selector.re('job', '($namespace)/cortex-gw')],
-                           queryFrontend: [utils.selector.re('job', '($namespace)/query-frontend')],
-                           querier: [utils.selector.re('job', '($namespace)/querier')],
-                           ingester: [utils.selector.re('job', '($namespace)/ingester')],
-                           querierOrIndexGateway: [utils.selector.re('job', '($namespace)/(querier|index-gateway)')],
+                           queryFrontend: [utils.selector.re('job', '($namespace)/%s' % (if !$._config.ssd then 'query-frontend' else '(enterprise-logs|loki)-read'))],
```

Comment:
same feedback here about asserting against the inverse of the condition, I think flipping the if/else would read a bit nicer.